_guides/amqp-guide.adoc

```diff
@@ -14,7 +14,7 @@ To complete this guide, you need:
 
 * less than 15 minutes
 * an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* {jdk-version} installed with `JAVA_HOME` configured appropriately
```

Comment:
I wonder if we should have includes containing the common Java/Maven/GraalVM requirements. I think it would be easier to maintain in the long run.